src/components/__tests__/codeBlock.test.js

```diff
@@ -1,13 +1,35 @@
 import React from 'react';
 import {create} from 'react-test-renderer';
 
+import { CodeBlock } from '../codeBlock';
 import {_setCachedCodeKeywords, CodeContextProvider, DEFAULTS} from '../codeContext';
 import {makeKeywordsClickable} from '../codeKeywords';
 
 function CodeWrapper({children, ...props}) {
   return <code {...props}>{children ? makeKeywordsClickable(children) : children}</code>;
 }
 
+// Tests that CodeBlock renders SignInNote when necessary.
+describe('CodeBlock', () => {
+  beforeEach(() => {
+    _setCachedCodeKeywords(DEFAULTS);
+  });
+
+  it('renders a SignInNote above the code block if the code has a placeholder', () => {
```

Comment:
This needs a mock for `SignInNote` because it's trying to use `data.site` from a static query that doesn't work. Send help — how do I best mock it?

Reply:
@natikgadzhi - left a comment above. LMK if you have other issues!